.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
```

Comment:
Given line 13, shouldn't this be something like "ensure file ends with a newline"?